ConanInstall.cmake

```diff
@@ -172,7 +172,7 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
     "jsoncpp/1.9.5#536d080aa154e5853332339bf576747c"
     "minizip/1.2.12#0b5296887a2558500d0323c6c94c8d02" # This depends on zlib, and basically patches it
     "zlib/1.2.12#3b9e037ae1c615d045a06c67d88491ae" # Also needed, so we can find zlib.h and co (+ pinning exactly is good)
-    "fmt/9.1.0#811e918ca4b4e0b9ddd6d5a2883efa82"
+    "fmt/8.1.1#b3e969f8561a85087bd0365c09bbf4fb"
```

Comment:
Uh oh, I see there's an issue with the git history here...

Reply:
@brianlball I see, you merged develop in then reverted that commit. I'm going to fix the branch, and rebase onto develop and force push. Make sure to update your local copy if you're going to work on it again.